Project1/Tile.cpp

```diff
@@ -0,0 +1,33 @@
+#include "Tile.h"
+Tile::Tile(): _unit(nullptr), _terrain() {
```

Comment:
members are initialized in the order they were declared in the h file, regardless of how you order them in constructor

it makes it confusing if you aren't consistent